Azure/communication-ui-library-android
# [Feature] OS Locale language support incase localization configuration is not provided

## Purpose

* We provide localization support even if the developer does not provide localization configuration settings
* In this case, we try to provide support for the OS Locale and if not we provide 'en_US' 

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test


* Change device language to anything other than English and among our supported language,. i.e. Espanol
* Comment out the compositeBuilder's setLocale from either Kotlin/Java launcher
* Deploy the app and launch the call to check if the language is Español or not.
* Now again test by setting a languag in device that is not supported by us.
* Now run the app and launch with the launcher which did not have the localizationConfiguration
* Check if the call setup window language is English or not.

## What to Check
Verify that the following are valid
* Check if testing property is valid

## Other Information

* The following PR solves [User Story 2812200](https://skype.visualstudio.com/SPOOL/_workitems/edit/2812200): As a Contoso user, the composite must detect my OS language to localize correctly the composite
